src/Sarif.Viewer.VisualStudio/Models/SarifErrorListItem.cs

```diff
@@ -76,20 +76,22 @@ public SarifErrorListItem(Run run, Result result, string logFilePath, ProjectNam
             Invocation = run.Invocations?[0]?.ToInvocationModel();
             WorkingDirectory = Path.Combine(Path.GetTempPath(), _runId.ToString());
 
-            if (result.Locations != null)
+            if (result.Locations?.Any() == true)
```

Comment:
>== true [](start = 40, length = 7)

you can remove direct comparison to boolean here and elsewhere in this change, doesn't materially improve readability

Reply:
No, because the left-hand side is a nullable bool (values true, false, or null) and so can't be implicitly converted to bool.